constraint/pkg/apis/templates/v1/defaults.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+import (
+	"github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates"
+	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema/defaulting"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const version = "v1"
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	return RegisterDefaults(scheme)
+}
+
+func SetDefaults_ConstraintTemplate(obj *ConstraintTemplate) {
+	// turn the CT into an unstructured
+	un, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+	if err != nil {
+		// JULIAN - what do I do here?  These functions don't return errors
```

Comment:
I think erroring is the best we can do here, unfortunately.

We should make sure this code path gets hit in a unit test to avoid errors, but these should never error, as any golang struct backing a resource should be convertible to unstructured.

Reply:
Removed comments.  Paths are already hit in tests.